app/assets/stylesheets/components/_booking_card.scss

```diff
@@ -1,4 +1,9 @@
 .booking-card-cust {
   width:auto;
+  height:fit-content;
+  padding:30px;
   box-shadow: 8px 6px 2px $black;
+  border-radius: 15px;
+  position:sticky;
+  top: 50px;
```

Comment:
styles booking card to display on right side of description, and sticks on scroll.